build.gradle

```diff
@@ -121,6 +121,8 @@ dependencies {
     coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.1.6'
 
     implementation 'androidx.appcompat:appcompat:1.5.0'
+    implementation 'androidx.lifecycle:lifecycle-viewmodel:2.5.1'
+    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.5.1'
```

Comment:
Why the ktx?

Reply:
It is the kotlin extension for the lifecycle-viewmodel. Somehow both libs are referenced in the new appcompat but with incompatible version, so the solution here is to override the versions to be the same.